### ISSUE TITLE
PPX suggestions

### ISSUE THREAD

**Issue by AngryLawyer** [fxfactorial/ocaml-reactjs#3]
So, I've kinda abandoned my existing pop at this at https://github.com/AngryLawyer/reactjs_of_ocaml, but I did get as far as writing a PPX for it.

With some changes to better match the style of element construction you've got going on, would you like me to investigate porting it to your project? Makes sense for something useful to come out of my failed attempt!

**Comment by fxfactorial** [comment]
Yes! I think it would be good to put all our collective knowledge and effort at one binding, and the commonjs was useful!!

**Comment by AngryLawyer** [comment]
Nice one. I'll start coming up with ideas in my free time.
